Dockerfile

```diff
@@ -4,3 +4,13 @@ MAINTAINER support@68publishers.io
 
 ########################################################################################################################
 FROM postgres:13.6-alpine AS db
+
+########################################################################################################################
+FROM 68publishers/docker-images:php-nginx-unit-7.4 AS worker
```

Comment:
We should use just base php-cli image here.

Reply:
I'm going to give it a try, anyway I did it this way because I didn't want to deal with PHP extensions separately.